revdep/problems.md

```diff
@@ -350,15858 +491,125 @@ Run `revdepcheck::cloud_details(, "andurinha")` for more info
-      `c1` not equal to c("gg", "ggplot").
-      Lengths differ: 5 is not 2
-      ── Failure ('test_misc-invgamma.R:35:3'): plotting the inverse gamma distribution works ──
-      `c2` not equal to c("gg", "ggplot").
-      Lengths differ: 5 is not 2
-      
-      [ FAIL 2 | WARN 1 | SKIP 0 | PASS 446 ]
-      Error: Test failures
-      Execution halted
-    ```
-
-## In both
-
-*   checking installed package size ... NOTE
-    ```
-      installed size is 153.2Mb
-      sub-directories of 1Mb or more:
-        R       1.5Mb
-        libs  151.3Mb
-    ```
-
-*   checking Rd files ... NOTE
-    ```
-    checkRd: (-1) prior_to_num.Rd:13: Lost braces; missing escapes or markup?
-        13 |   {'uniform', 'normal', 'student-t', 'gamma', 'inv-gamma', 'log-normal'}
-           |   ^
-    ```
-
-*   checking for GNU extensions in Makefiles ... NOTE
-    ```
-    GNU make is a SystemRequirements.
-    ```
-
-# listdown
-
-<details>
-
-* Version: 0.5.7
-* GitHub: https://github.com/kaneplusplus/listdown
-* Source code: https://github.com/cran/listdown
-* Date/Publication: 2023-04-03 00:30:02 UTC
-* Number of recursive dependencies: 70
-
-Run `revdepcheck::cloud_details(, "listdown")` for more info
-
-</details>
-
-## Newly broken
-
-*   checking examples ... ERROR
-    ```
-    Running examples in ‘listdown-Ex.R’ failed
-    The error most likely occurred in:
-    
-    > ### Name: as_ld_yml
-    > ### Title: Turn a Computational Component List into YAML with Class
-    > ###   Information
-    > ### Aliases: as_ld_yml
-    > 
-    > ### ** Examples
-    > 
-    ...
-    +   as_ld_yml(cc_list)
-    + }
-    Loading required package: ggplot2
-    Error in parse(text = paste("x", loc, " <<- list(paste(", deparse(class(elem)),  : 
-      <text>:2:1: unexpected symbol
-    1: x [[1]]  <<- list(paste( c("ggplot2::ggplot", "ggplot", "ggplot2::gg", "S7_object", "gg" , collapse = ":"))
-    2: x
-       ^
-    Calls: as_ld_yml -> depth_first_copy -> eval -> parse
-    Execution halted
-    ```
-
-*   checking tests ... ERROR
-    ```
-      Running ‘testthat.r’
-    Running the tests in ‘tests/testthat.r’ failed.
-    Complete output:
-      > library(testthat)
-      > library(listdown)
-      > 
-      > test_check("listdown")
-      [ FAIL 2 | WARN 0 | SKIP 1 | PASS 34 ]
-      
-      ══ Skipped tests (1) ═══════════════════════════════════════════════════════════
-    ...
-       5.   └─listdown (local) depth_first_copy()
-       6.     ├─base::eval(...)
-       7.     └─base::parse(...)
-      ── Failure ('test-cc-dendro.r:20:3'): Dendrograms work. ────────────────────────
-      ld_cc_dendro(cc_list) not equal to read_reference("cc-dendro.rds").
-      4 string mismatches
-      
-      [ FAIL 2 | WARN 0 | SKIP 1 | PASS 34 ]
-      Error: Test failures
-      Execution halted
-    ```
-
-## In both
-
-*   checking package dependencies ... NOTE
-    ```
-    Package which this enhances but 
```

Comment:
This is my bad, should have fallback for missing parameter in `StatAlign$finish_layer()`